### ISSUE TITLE
flowcell_map.py is trying to create a file in raw data folder 

### ISSUE THREAD

**Issue by NmnBttr** [rajewsky-lab/openst#67]
Hi,

Thanks for the great tool! 
I tried running the preprocessing capture area library script on a HPC. 
This is how I called the script:
```sh
openst flowcell_map \
    --bcl-in $BCL_DIR \
    --tiles-out $WORK_DIR \
    --crop-seq 5:30 \ 
    --rev-comp
```

And this is the Error I am encountering: 
```sh
  File "/path.../openst/preprocessing/flowcell_map.py", line 396, in _run_flowcell_map
    with open(lanes_and_tiles_path, "w") as f:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
PermissionError: [Errno 13] Permission denied: '/path/to/$BCL_DIR/lanes_and_tiles.txt'
```
The problem here seems to be the code at line 396 in **flowcell_map.py** which tries to create the _lanes_and_tiles.txt_ in the raw data folder containing the bcl files. This is not ideal if working on a HPC where one only has the "read" privileges. Hence, is it possible to fix this by creating the _lanes_and_tiles.txt_ file inside the given output folder, and also in general to not "write" files in the raw data directory?

Thank you in advance!

## Environment info
- Platform: Linux
- Python version: 3.11
- openst version: 0.2.1

**Comment by danilexn** [comment]
Hi! Thanks for the nice suggestion :) 

This is fixed now in [07fe289](https://github.com/rajewsky-lab/openst/commit/07fe2896ac25d3dbc11e0c40e642bfa6738f93a9)

Will make the new version available in pip shortly

**Comment by NmnBttr** [comment]
Perfect! Thanks and looking forward to it!
